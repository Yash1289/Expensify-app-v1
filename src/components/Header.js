const React = require("react");
import { Link , NavLink} from "react-router-dom"

const Header = () => (
    <div>
        <h1>Expensify App</h1>
        <NavLink to="/" activeClassName="is-active" exact={true}>Home</NavLink>
        <br></br>
        <NavLink to="/create" activeClassName = "is-active">Create</NavLink>
        <br></br>
        <NavLink to="/help" activeClassName = "is-active">Help</NavLink>
    </div>
)

export default Header;