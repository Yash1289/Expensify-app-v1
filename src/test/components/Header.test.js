import { TestScheduler } from "jest"
import React from "react"
import { shallow } from "enzyme"
import ReactShallowRenderer from "react-test-renderer/shallow"
import Header from "../../components/Header"


test("Should render header component ", () => {
    const wrapper = shallow(<Header />)
    expect(wrapper).toMatchSnapshot(); 
      /*  const renderer = new ReactShallowRenderer();
    renderer.render(<Header />)
    expect(renderer.getRenderOutput()).toMatchSnapshot();     */
})

