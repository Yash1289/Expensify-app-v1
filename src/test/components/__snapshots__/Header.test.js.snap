// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render header component  1`] = `
<div>
  <h1>
    Expensify App
  </h1>
  <[object Object]
    activeClassName="is-active"
    exact={true}
    to="/"
  >
    Home
  </[object Object]>
  <br />
  <[object Object]
    activeClassName="is-active"
    to="/create"
  >
    Create
  </[object Object]>
  <br />
  <[object Object]
    activeClassName="is-active"
    to="/help"
  >
    Help
  </[object Object]>
</div>
`;
